dotnet-base-libraries/ReturnInsertedIdentityWithDapper/ReturnInsertedIdentityWithDapper/Methods/Methods.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Dapper;
+using Microsoft.Data.SqlClient;
+
+namespace ReturnInsertedIdentityWithDapper;
+
+public class Methods
```

Comment:
Should be static as there is no state, only static methods

Reply:
Converted to static